Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsControl.resx

```diff
@@ -520,10 +298,10 @@
     <value>GrowAndShrink</value>
   </data>
   <data name="$this.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>6, 6, 0, 6</value>
+    <value>3, 3, 0, 3</value>
   </data>
   <data name="$this.Size" type="System.Drawing.Size, System.Drawing">
-    <value>735, 745</value>
+    <value>368, 260</value>
```

Comment:
Here's VS14:

![image](https://cloud.githubusercontent.com/assets/12821956/18399376/6d6ab4da-7685-11e6-9934-53a20641236d.png)

In VS15, the page goes away entirely.